### PR TITLE
New serverless pattern - CloudFront to Lambda Function URL using OAC

### PR DESCRIPTION
**Pull request by sshekghub**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cloudfront-lambdafunctionurl-oac-sam/LICENSE
Comment (marcojahn) [inline cloudfront-lambdafunctionurl-oac-sam/LICENSE:1]: File can be removed

File: cloudfront-lambdafunctionurl-oac-sam/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Access your Lambda Function URLs securely using Amazon CloudFront Origin Access Control
```
Comment (marcojahn) [inline cloudfront-lambdafunctionurl-oac-sam/README.md:1]:
```suggestion
# Amazon CloudFront Origin Access Control for AWS Lambda Function URL
```

File: cloudfront-lambdafunctionurl-oac-sam/example-pattern.json
Comment (marcojahn) [inline cloudfront-lambdafunctionurl-oac-sam/example-pattern.json:1]: Getting multiple json parser errors, can you please check and fix?

File: cloudfront-lambdafunctionurl-oac-sam/example-pattern.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{
+  "title": "Access your Lambda Function URLs securely using Amazon CloudFront Origin Access Control",
```
Comment (marcojahn) [inline cloudfront-lambdafunctionurl-oac-sam/example-pattern.json:2]:
```suggestion
  "title": "Amazon CloudFront Origin Access Control for AWS Lambda Function URLl",
```

**Review by marcojahn — CHANGES_REQUESTED** [review]

**Comment by sshekghub** [comment]
Made the requested changes 1/ Removed License file 2/ Updated title in README as well as in pattern json file 3/ Fixed errors in pattern json file.

**Review by marcojahn — APPROVED** [review]

**Comment by marcojahn** [comment]
Hello @sshekghub thank you for your contribution. The pattern will be merged to serverlessland.com soon.

**Pull request merged**